### DISCUSSION TITLE
Nhận task

### DISCUSSION THREAD

**Discussion by trungdoublelift** [Runway-Club/Runway-CV-Builder#14] [Task Delivery]
Tôi muốn nhận 2 task là thiết kế giao diện CV list, giao diện xem danh sách CV

**Comment by nguyentantrung2000** [comment]
Sau khi xem qua thì 2 task này chưa có người làm. Thì mình sẽ đưa cho bạn 2 task này. 
Một số thông tin của task:
1. Là các trang chủ đạo của trang web.
2. Sẽ hơi phức tạp đối với những người mới.
3. Phải biết về Reactjs trước khi làm.
Yêu cầu:
1. Màu sắc phải đúng như trong figma(các màu chủ đạo)
2. Đúng các bố cục figma
3. Có thể dùng các nút để lướt qua lại xem CV 
4. deadline 11:59 PM 30/7/2021

**Reply by trungdoublelift** [reply]
OK
